nucypher/cli/types.py

```diff
@@ -27,7 +27,12 @@ class ChecksumAddress(click.ParamType):
     name = 'checksum_address'
 
     def convert(self, value, param, ctx):
-        return to_checksum_address(value=value)  # TODO: More robust validation here?
+        try:
+            value = to_checksum_address(value=value)
+        except ValueError as e:
+            self.fail("Invalid ethereum address")
```

Comment:
Nice to know what the problem was:

```suggestion
            self.fail(f"Invalid ethereum address: {str(e)}")
```